tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -7,8 +7,16 @@
 #include "abbrev.h"
 #include "cmontecarlo.h"
 #include "io.h"
-
-
+long long int count_resonance = 0;
+static int
+compare(const void *ap, const void *bp)
+{
```

Comment:
@SooryaN - if you are using the current line interaction and resonance determination infrastructure of the code, you will never have to compare frequencies actually.